packages/kujua-sms/tests/kujua-sms/utils.js

```diff
@@ -160,294 +162,6 @@ exports['getLabels'] = function(test) {
     test.done();
 };
 
-exports['getValues no clinic'] = function(test) {
```

Comment:
I did delete the function too...

I don't think it's useful, and we can always get it back out of git if I'm wrong.